deploy/cloudformation/elastic-agent-ec2-cnvm.yml

```diff
@@ -130,12 +133,15 @@ Resources:
         - !Ref ElasticAgentSecurityGroup
       UserData: !Base64
         Fn::Sub: |
-          #!/bin/bash
+          #!/bin/bash -x
```

Comment:
Branch is based on top of https://github.com/elastic/cloudbeat/pull/1163, when that is merged this diff will disappear